app/views/practices/implementation_forms/_risks_and_mitigations.html.erb

```diff
@@ -2,6 +2,47 @@
   <%= javascript_include_tag 'practice_editor_risk_mitigation', 'data-turbolinks-track': 'reload' %>
   <%= javascript_include_tag '_practice_editor_utilities', 'data-turbolinks-track': 'reload' %>
 <% end %>
+<% risk_join_array = Array.new %>
```

Comment:
I was thinking we would delete risks/mitigations without a corresponding mitigation/risk via a rake task not in the view since this is a one-time data issue that needs correcting.

Reply:
I agree... you mentioned this in planning... and I agreed with it then as well.  I'll move the code to a rake task.